docs/reference-architectures/sharepoint/index.md

```diff
@@ -111,143 +116,264 @@ Before configuring the SharePoint farm, make sure you have one Windows Server AD
+
+The default service applications in this deployment are designed to support hybrid workloads. All SharePoint Server 2016 and Office 365 hybrid workloads can be deployed to this farm without changes to the SharePoint infrastructure—with one exception. The Cloud Hybrid Search Service Application must not be deployed onto servers hosting an existing search topology. Therefore, one or more search-role-based virtual machines must be added to the farm to support this hybrid scenario. 
 
 ## Security considerations
+The domain-level service accounts used to run SharePoint Server 2016 require Windows Server AD domain controllers for domain-join and authentication processes. Azure Active Directory Domain Services can’t be used instead. To extend the Windows Server AD identity infrastructure already in place in the intranet, this architecture uses two Windows Server AD replica domain controllers of an existing on-premises Windows Server AD forest. See also [Create an AD DS resource forest in Azure][ad-ds].
+
+In addition, it's always wise to plan for [security hardening][security-hardening]. Other recommendations include:
 
-The domain-level service accounts used to run SharePoint Server 2016 require Windows Server AD domain controllers for domain-join and authentication processes. Azure Active Directory Domain Services can't be used for this purpose. To extend the Windows Server AD identity infrastructure already in place in the intranet, this architecture uses two Windows Server AD replica domain controllers of an existing on-premises Windows Server AD forest.
+- Add rules to network security groups to isolate subnets and roles.
 
-In addition, it's always wise to plan for security hardening. Other recommendations include:
+- Don’t assign public IP addresses to VMs.
 
-- Add rules to NSGs to isolate subnets and roles.
-- Don't assign public IP addresses to VMs.
 - For intrusion detection and analysis of payloads, consider using a network virtual appliance in front of the front-end web servers instead of an internal Azure load balancer.
+
 - As an option, use IPsec policies for encryption of cleartext traffic between servers. If you are also doing subnet isolation, update your network security group rules to allow IPsec traffic.
+
 - Install anti-malware agents for the VMs.
 
-## Deploy the solution
+## Community
+Communities can answer questions and help you set up a successful deployment. Consider the following:
+
+- [Azure Forum][azure-forum]
+
+- [TechNet Wiki: SharePoint: Community Best Practices][technet-wiki]
+
+- [SharePoint Microsoft Tech Community][sharepoint-community]
+
+## Solution deployment
+
+The deployment scripts for this reference architecture are available on [Github][github]. To run the PowerShell script that deploys this architecture, use the latest version of the Azure [command line interface][azure-cli] (CLI).
```

Comment:
Doesn't make sense.   It should be something like this  ".... use the latest version of PowerShell for Azure".